external/model-preparation-algorithm/mpa_tasks/apis/classification/task.py

```diff
@@ -158,6 +159,35 @@ def infer(
         self._add_predictions_to_dataset(prediction_results, dataset, update_progress_callback)
         return dataset
 
+    def explain(
+        self,
+        dataset: DatasetEntity,
+        explain_parameters: Optional[InferenceParameters] = None,
+    ) -> DatasetEntity:
+        logger.info("called explain()")
+        stage_module = "ClsExplainer"
+        self._data_cfg = self._init_test_data_cfg(dataset)
+        dataset = dataset.with_empty_annotations()
+
+        results = self._run_task(
+            stage_module,
+            mode="train",
+            dataset=dataset,
+            explainer=explain_parameters.explainer,
+        )
+        logger.debug(f"result of run_task {stage_module} module = {results}")
+        outputs = results["outputs"]
+        saliency_maps = zip(
+            outputs["saliency_maps"],
+        )
```

Comment:
`outputs["saliency_maps"]` is a list. No need to `zip` it. You can replace with just
`saliency_maps = results["outputs"]["saliency_maps"]`

Reply:
fixed